sdkgen/nodejs-request/lib/util.js

```diff
@@ -0,0 +1,81 @@
+const codegen = require('postman-code-generators'),
+  sdk = require('postman-collection');
+
+/**
+ * [Description]
+ *
+ * @param {String} requestSnippet - Request snipept generated by postman-code-generator
+ * @returns {String} - Request snippet string with replaced collection variables
+ */
+function replaceVariables (requestSnippet) {
+  var variableDeclarations = requestSnippet.match(/{{.*}}/g);
+  if (variableDeclarations) {
+    variableDeclarations.forEach((element) => {
+      // replacing {{variable_name}} with ' + this.variables.variable_name + '
+      requestSnippet = requestSnippet
+        .replace(element, '\' + self.variables.' + element.substring(2, element.length - 2) + ' + \'');
```

Comment:
What happens if the variables are at the end or beginning?

Reply:
say we have `'{{{base_url}}/home'` then it will be replaced as `'' + self.variables.base_url + '/home'` , same case when its in the end, it will be just an `''` at the end